recipes/epysurv/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "epysurv" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  url: https://github.com/jarnorfb/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 56383ce024e2ad76c1ac300065efe555cc57fcbc8d7ab00052b310195470e80c
+
+build:
+  # noarch: python
+  number: 0
+  merge_build_host: True  # [win]
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+  host:
+    - python=3.7
+    - pip
+    - r-base=3.5
```

Comment:
IDK if pinning r-base is a great idea.

Reply:
Agreed. r-base should not be pinned without good reason. Does this package not work with R 3.4 or 3.6?

Reply:
I changed it to r-base >=3.2 which is what it says in the R surveillance package documentation.

Reply:
@JarnoRFB You don't need to pin r-base at all. `conda-build` handles r-base differently than the other packages

Reply:
@jdblischak ok I removed the pin